README.md

```diff
@@ -73,3 +73,30 @@ Note: If you are using more then one type of component style files (ie a .less f
  - [`ember-cli-sass`](https://github.com/aexmachina/ember-cli-sass)
  - [`ember-cli-less`](https://github.com/gdub22/ember-cli-less)
  - [`ember-cli-stylus`](https://github.com/drewcovi/ember-cli-stylus)
+
+### Configuration
+
+You can set the following configuration options in your `config/environment.js` file:
+
+```js
+ENV['ember-component-css'] = {
+  option: 'value'
+}
+```
+
+**namespacing(_enabled_)**
+
+Defaults to true. Set this option to `false` to disable the namespacing feature of Ember Component CSS.
+
+```js
+ENV['ember-component-css'] = {
+  namespacing: false
+}
+```
+
+If `false`, the default behavior changes in two ways:
+
+ 1. The autogenerated component id is no longer added to your component's HTML
```

Comment:
can we change these references to id's to classnames so people don't get confused thinking that we are tweaking with id's?